tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py

```diff
@@ -6,6 +6,9 @@
 from ...row_schema import RowSchema, SchemaManager
 from ... import validators
 from tdpservice.search_indexes.documents.ssp import SSP_M6DataSubmissionDocument
+import datetime
+
+minYear = datetime.date.today().year - 5
```

Comment:
@atrimpe I don't think we need the minyear to be set relative to today's date, since the system is not set up to accept fiscal periods prior to FY2021. i believe this setting was implemented [here](https://github.com/raft-tech/TANF-app/blob/fd07182006adac14b1b7fcb291c003970f800b16/tdrs-frontend/src/components/Reports/Reports.jsx#L173), and OFA is not currently allowing data submissions prior to FY21 (this may change in the future)

Reply:
Ok, i was requested to add this in raft review. I will remove. 👍